wallet/vault/vault_test.go

```diff
@@ -231,6 +231,27 @@ func TestNewBLSAccountAddress(t *testing.T) {
 	assert.Equal(t, label, addressInfo.Label)
 }
 
+func TestNewED25519AccountAddress(t *testing.T) {
+	td := setup(t)
+
+	t.Run("Ok", func(t *testing.T) {
+		addressInfo, err := td.vault.NewEd25519AccountAddress("addr-1", tPassword)
+		assert.NoError(t, err)
+		assert.NotEmpty(t, addressInfo.Address)
+		assert.NotEmpty(t, addressInfo.PublicKey)
```

Comment:
Check Path: assert.Equal(t, "m/44'/21888'/3'/0'", addressInfo.Path)

Same for others